css/project-page.styl

```diff
@@ -1,3 +1,21 @@
+.about-container
```

Comment:
At some point this should be redone with BEM since that seems to be the direction we're going for this repo, but could you follow the convention that is in the rest of this stylesheet for now? Perhaps `project-about-page` instead?